landlab/components/landslides/landslide_probability.py

```diff
@@ -441,32 +500,70 @@ def __init__(
             self._recharge_mean = groundwater__recharge_mean
             self._recharge_stdev = groundwater__recharge_standard_deviation
-        # Custom HSD inputs - Hydrologic Source Domain -> Model Domain
+        # Recharge Custom HSD inputs - Hydrologic Source Domain -> Model Domain
         elif self._groundwater__recharge_distribution == "data_driven_spatial":
             self._HSD_dict = groundwater__recharge_HSD_inputs[0]
             self._HSD_id_dict = groundwater__recharge_HSD_inputs[1]
             self._fract_dict = groundwater__recharge_HSD_inputs[2]
             self._interpolate_HSD_dict()
 
+        # Following code will deal with the input distribution and associated
+        # parameters for depth to water table hydrologic forcing
+        # Depth to water table - Uniform distribution
+        if self._groundwater__depth_distribution == "uniform":
+            self._depth_min = groundwater__depth_min_value
+            self._depth_max = groundwater__depth_max_value
+            self._De = np.random.uniform(self._depth_min, self._depth_max, size=self._n)
+        # Depth to water table - Lognormal Distribution - Uniform in space
+        elif self._groundwater__depth_distribution == "lognormal":
+            assert (
+                groundwater__depth_mean is not None
+            ), "Input mean of the distribution!"
+            assert (
+                groundwater__depth_standard_deviation is not None
+            ), "Input standard deviation of the distribution!"
+            self._depth_mean = groundwater__depth_mean
+            self._depth_stdev = groundwater__depth_standard_deviation
+            self._mu_lognormal = np.log(
+                (self._depth_mean ** 2)
+                / np.sqrt(self._depth_stdev ** 2 + self._depth_mean ** 2)
+            )
+            self._sigma_lognormal = np.sqrt(
+                np.log((self._depth_stdev ** 2) / (self._depth_mean ** 2) + 1)
+            )
+            self._De = np.random.lognormal(
+                self._mu_lognormal, self._sigma_lognormal, self._n
+            )
+
+        # Depth to water table - Lognormal Distribution - Variable in space
+        elif self._groundwater__depth_distribution == "lognormal_spatial":
+
+            # assert groundwater__depth_mean.shape[0] == (
+            #    self._grid.number_of_nodes
+            # ), "Input array should be of the length of grid.number_of_nodes!"
+            # assert (groundwater__depth_standard_deviation.shape[0] == (
+            #    self._grid.number_of_nodes
+            # ), "Input array should be of the length of grid.number_of_nodes!"
+            self._depth_mean = groundwater__depth_mean
+            self._depth_stdev = groundwater__depth_standard_deviation
+
+        # Depth to water table - Hydrologic Source Domain -> Model Domain
+        elif self._groundwater__depth_distribution == "data_driven_spatial":
+            self._HSD_dict = groundwater__depth_HSD_inputs
+
```

Comment:
It appears that there is no checking if the inputs are overspecified. For example if someone wrote: 

groundwater__recharge_distribution='uniform'
and 
groundwater__depth_distribution='uniform'

what would happen? 

Recommend having a bit more checking in the init, and add a few sentences to the docs describing the precedence/over specified input checking.